shared/common-adapters/plain-input.js.flow

```diff
@@ -0,0 +1,86 @@
+// @flow
```

Comment:
This file and `.shared.js` deliberately copy a lot from `input.*` so it doesn't introduce any dependency for when we do a more full transition